src/sql/platform/accounts/test/common/accountStore.test.ts

```diff
@@ -327,6 +329,19 @@ suite('Account Store Tests', () => {
 			});
 	});
 
+	test('Remove - Deprecated Account', () => {
+		let memento = getTestMemento();
+		memento[AccountStore.MEMENTO_KEY].push(deprecatedAccount1, deprecatedAccount2);
+		let as = new AccountStore(memento, consoleLogService);
+		// We know that we have 4 accounts now
+		assert.equal(memento[AccountStore.MEMENTO_KEY].length, 4);
+
+		return as.getAllAccounts().then(accounts => {
```

Comment:
You need to pass in the done callback here otherwise the assert is not going to fail correctly

Reply:
Oh do the other tests have the same issue?

Reply:
Oh hm, yeah you're right. Since you're returning the thenable it waits on that correctly. And actually reading the docs calling done would result in an error. So yeah looks fine.